tests/project_directory_test.php

```diff
@@ -107,4 +107,20 @@ public function testGetDirectoryAvoidPluginsDirectory()
         $this->assertInstanceOf('\\Dxw\\Whippet\\ProjectDirectory', $result->unwrap());
         $this->assertEquals($dir, $result->unwrap()->__toString());
     }
+
+    public function testGetDirectoryWithEnv()
+    {
+        $dir = $this->getDir();
+
+        putenv('WHIPPET_PROJECT_DIRECTORY=/etc');
```

Comment:
Do we need to clean up afterwards? Will this value be set for all subsequent tests?

Is there any risk that we could end up writing test data into `/etc` which we leave lying around after the test suite run?